analysis/webservice/algorithms/doms/subsetter.py

```diff
@@ -203,14 +203,22 @@ def calc(self, request, **args):
 
             tile = tile[0]
 
+            def get_best_name(variable):
+                if variable.standard_name:
+                    return variable.standard_name
+                elif variable.variable_name:
+                    return variable.variable_name
+                else:
+                    return 'UNNAMED_VARIABLE'
```

Comment:
I wonder if we'd want to raise an exception here? I can't imagine reaching this state

Reply:
Done. Raises `NexusProcessingException` w/ code 500